cmd/manager/main.go

```diff
@@ -204,3 +218,30 @@ func serveCRMetrics(cfg *rest.Config) error {
 	return nil
 }
 */
+func IsTopologyInUse(ctx context.Context) (bool, error) {
+	kubeConfig, err := rest.InClusterConfig()
+	if err != nil {
+		log.Info("unable to load in-cluster configuration: %v", err)
+		log.Info("skipping topology retrieval. we might not be in a k8s cluster")
```

Comment:
Why the first log's severity is `info`? Shouldn't it be higher?
Can we merge these messages together?

Reply:
it will log with `log.Error` in the parent method.
do you think it's problematic as two separated logs?

Reply:
Why do we need three messages for one error?
Don't we prefer one message (maybe longer)?

Reply:
I don't think the operator can function if it can't get the kube config.
this should be more severe than "info", and shouldn't assume `we might not be in a k8s cluster` (in contrast to the driver), because we don't plan to ever support such scenario for this operator

Reply:
ok, true
removed logs there.